### PR TITLE
Issue #377 Remove nav on questionnaire

### PR DESCRIPTION
**Pull request by bensinjin**
Note: The point about making the bookmark button larger was done under #369 (pr #388)

### CODE REVIEW

File: src/components/main/footer.tsx [hunk 1]
```diff
@@ -13,11 +13,16 @@ export interface FooterProps {
...
     const path = props.location.pathname;
+    const isOnQuestionnaireScreen = pathMatchesRoute(path, Routes.Questionnaire);
 
     if (isOnStartScreen(path)) {
         return <EmptyComponent />;
     }
 
+    if (isOnQuestionnaireScreen) {
+        return <EmptyComponent />;
+    }
```
Comment (rasmus-storjohann-PG) [inline src/components/main/footer.tsx]: Inconsistent with the code directly above

Comment (bensinjin) [reply src/components/main/footer.tsx]: I've used this pattern for the header so keeping it consistent. I wasn't planning on making an `isOnXScreen` for every path as I've previously stated when this has been brought up. `isOnStartScreen` `isOnParentScreen` and `isOnChildScreen` are meant to be more general.

Comment (rasmus-storjohann-PG) [reply src/components/main/footer.tsx]: I agree, I'm not a fan of `isOnStartScreen()`, so I would take that out in this PR.

Comment (bensinjin) [reply src/components/main/footer.tsx]: Ah, gotcha, fixed

**Pull request merged**